buildpacks/python.go

```diff
@@ -139,6 +139,12 @@ func (bt PythonBuildTool) InstallPlatformDependencies() error {
 				cmd.Run()
 			}
 		}
+		// Thanks to henriquebastos, here:
+		// https://github.com/pyenv/pyenv/issues/1066#issuecomment-504710495
+		if strings.HasPrefix(gi.Core, "18.6") {
```

Comment:
I feel like maybe these are things lots of build packs will need... 🤔 but it's fine here for now